Bottom: c395e3da3639d7a6ca503d0910bcba8af2e597d5
Top:    602178c30a5fa740ed32810ec59eec33c398d8ef
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 23:05:09 +0000

hid/gerber: Fix a couple of typos in drill_sort routine

Should fix sorting of drill holes by coordinate.

(Caught by clang)


---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index c0faf47..9579b7b 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -631,8 +631,8 @@ drill_sort (const void *va, const void *vb)
   if (a->diam != b->diam)
     return a->diam - b->diam;
   if (a->x != b->x)
-    return a->x - a->x;
-  return b->y - b->y;
+    return a->x - b->x;
+  return a->y - b->y;
 }
 
 static int
